internal/daysteps/daysteps.go

```diff
@@ -12,9 +18,58 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	parsedData := strings.Split(data, ",")
+	if len(parsedData) != 2 {
+		return 0, 0, errors.New("")
+	}
+	stepsSrt := parsedData[0]
+	dateStr := parsedData[1]
+
+	steps, err := strconv.Atoi(stepsSrt)
+	if err != nil {
+		return 0, 0, err
```

Comment:
Лучше оборачивать ошибки
```suggestion
		return 0, 0, fmt.Errorf("invalid steps format: %w", err)
```